### PR TITLE
feat: enable attributes / personal search on OBF, OPF, OPFF, new config + logos for OPF & OPFF

### PR DESCRIPTION
**Pull request by stephanegigandet**
- Some changes to enable knowledge panels on OBF, OPF and OPFF
- new conf files for new deployments of OPF and OPFF
- new logos for OPF and OPFF

Test deployments (with the production databases):

https://world.new.openproductsfacts.org
https://world.new.openpetfoodfacts.org

### CODE REVIEW

**Comment by teolemon** [comment]
- [ ] Disabling Food specific facets

image

**Comment by teolemon** [comment]
The "Get the app" links are not working: `https://world.openproductsfacts.org/?utm_source=off&utf_medium=web&utm_campaign=search_and_links_promo_en`

**Comment by teolemon** [comment]
Disable Food specific search criterion
image

**Comment by teolemon** [comment]
image
Redirecting to https://wiki.openfoodfacts.org/Open_Products_Facts for the wiki (nice to have)

**Comment by teolemon** [comment]
Fixing the data (and content pages) which do not use the right folder from openfoodfacts-web: https://world.openproductsfacts.org/data

**Review by alexgarel — APPROVED** [review]
Great, only minor comments !

File: html/images/favicon/readme.md
[the remark this answers is not in the document]
Comment (stephanegigandet) [reply html/images/favicon/readme.md:1]: well that way we remember how we created them if we need to recreate them

File: lib/ProductOpener/KnowledgePanels.pm [hunk 1]
```diff
@@ -780,22 +793,111 @@ sub create_environment_card_panel ($product_ref, $target_lc, $target_cc, $option
...
+
+	# Find the first category that has a carbon_impact_fr_impactco2:en: property
+	my ($value, $category_id)
+		= get_inherited_property_from_categories_tags($product_ref, "carbon_impact_fr_impactco2:en");
+
+	$log->debug("create carbon footprint panel",
+		{code => $product_ref->{code}, category_id => $category_id, value => $value})
+		if $log->is_debug();
+
+	if ($value) {
```
[the remark this answers is not in the document]
Comment (stephanegigandet) [reply lib/ProductOpener/KnowledgePanels.pm:883]: indeed, thanks

File: templates/api/knowledge-panels/environment/carbon_footprint_product.tt.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        "subtitle": "[% lang("average_for_the_category").ucfirst %][% sep %]: [% panel.category_name %]",
+        "icon_url": "[% static_subdomain %]/images/panels/fr-impact-co2/impact-co2.svg",
+    },
+    "elements": [
+        {
+            "element_type": "text",
+            "text_element": {
+                "type": "summary",
+                "html": `
+                    <p>[% edq(f_lang('f_equal_to_driving_km_in_a_petrol_car',  { 'kilometers' => driving_per_unit_rounded } )) %]</p>
```
[the remark this answers is not in the document]
Comment (stephanegigandet) [reply templates/api/knowledge-panels/environment/carbon_footprint_product.tt.json:21]: @alexgarel it's not in normal JSON, it's to ease the creation of JSON for knowledge panels. It also handles the escaping of quotes that may be inside.

See https://github.com/openfoodfacts/openfoodfacts-server/blob/main/lib/ProductOpener/KnowledgePanels.pm#L374

**Comment by teolemon** [comment]
Given how large it is, we should tweak the wording of the donation message
image

**Comment by teolemon** [comment]
We should not link to https://world.pro.openproductsfacts.org/ which gives a HTTPS error.

**Comment by stephanegigandet** [comment]
@teolemon as this is in production already, I will go ahead and merge the PR, and I'll address the issues you raised in new PRs.

**Pull request merged**